workflowitems.md

```diff
@@ -116,6 +117,13 @@ The submission definition used by the inprogress submission is derived from the
 Please note that this endpoint is not strictly necessary as you can currently retrieve the same definition using [/api/config/submissiondefinitions/search/findByCollection?uuid=<:workspaceitem-collection-uuid>](submissiondefinitions.md#findByCollection) but it allows the client to find the submissionDefinition embedded in the workspaceitem without the need to make a separate call. 
 In addition, it allows in future to change the 1:1 association between collections and submissionDefinition without breaking the client
 
+#### workflow step
+**/api/workflow/claimedtasks/<:id>/step** (READ-ONLY)
+
+It returns the workflow step currently assigned to the task.
```

Comment:
A few copy & paste errors here.  Endpoint should be `/api/submission/workflowitems/<:id>/step` and the description something like "It returns the workflow step of the workflow item"

Reply:
Thanks for the review, this has been updated